lib/app_view.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/widgets/navigation_bar/navigation_bar.dart';
```

Comment:
we should note somewhere that we're using absolute paths so we can keep consistent with everyone

Reply:
I added a note in the `main.dart` file, which felt like the best place to put it.